### ISSUE TITLE
Insights into HPO for DBpedia50 datasets on KGE models

### ISSUE THREAD

**Issue by debanjali05** [nicolas-hbt/similar-embeddings#1]
Hi Nicholas, 

I am a Ph.D. student from GESIS under the supervision of Prof. Dr. Stefan Dietze. I am working with KGE models and wanted to know what kind of hyperparameter space you chose for all the KGE models trained on DBpedia50 in this GitHub repo.

I would really appreciate it if you could share some insights into this matter. I am using the LibKGE library to train 5 KGE models (TransE, DistMult, ComplEx, ConvE, and RESCAL) on the Dbpedia50 but for some reason, the HPO is not leading to a good performance. Your insights into the hyperparameter space for these models on this dataset using Pykeen could help me reproduce similar results as in your paper. 

I'm looking forward to hearing back from you! Thanks in advance! 😊

Best Regards, 
Debanjali

**Comment by nicolas-hbt** [comment]
Hi Debanjali,

Thanks for your message and your interest in our work. 
I have taken a brief look at my archives but I cannot find the chosen hyperparameters.
However, I suggest you take a look at the following two links:

1) https://github.com/nicolas-hbt/semantic-lossfunc/
2) https://github.com/nicolas-hbt/versatile-embeddings/

In particular, the README file will give you hyperparameters I used for DB77k. It is highly likely the one that worked best for DBpedia50 were in a close range.

If I manage to get access to my university clusters and if my files were not deleted in the meantime, perhaps I can get back to you with the exact hyperparameters. But as I already defended my PhD a few months ago, I might not be able to retrieve them.

Best regards,
Nicolas
